packages/eslint-plugin/src/rules/ban-types.ts

```diff
@@ -3,16 +3,7 @@ import { AST_NODE_TYPES } from '@typescript-eslint/utils';
 
 import * as util from '../util';
 
-type Types = Record<
-  string,
-  | null
-  | false
-  | string
-  | {
-      message: string;
-      fixWith?: string;
-    }
->;
```

Comment:
I'm not sure I understand - how come you changed this type here? It doesn't seem like this was required for your PR?

Reply:
I understood that changing record <string, never> to object. So I changed it to types and used it.

Reply:
I modified The type types back to record<>.